### ISSUE TITLE
Debug logs on ingest can lead to recursion

### ISSUE THREAD

**Issue by bahlo** [axiomhq/axiom-py#23]
From Christian on AxiomFM slack:

> I'm using axiom-py (version 0.1.0b2) in my Python backend. If I configure the default python logger to log to Axiom server and I  set the logging Level to DEBUG, I'm getting a recursion in log calls. This is because each axiom ingest http post request itself triggers a debug log event. Any ideas how to prevent this?

> This is my logging setup:
```py
import logging
import queue
from logging.handlers import QueueHandler, QueueListener
from axiom import Client

(...)

def _add_axiom_handler(logger: logging.Logger, formatter: logging.Formatter, dataset: str, app: Sanic = None):
    
    que = queue.Queue(-1)  # no limit on size
    queue_handler = QueueHandler(que)
    queue_handler.setLevel(settings.AXIOM_LOGGING_LEVEL)

    axiom_client = Client(url_base=settings.AXIOM_BASE_URL, token=settings.AXIOM_TOKEN)
    axiom_handler = AxiomHandler(client=axiom_client, dataset=dataset)

    listener = QueueListener(que, axiom_handler)
    logger.addHandler(queue_handler)
    axiom_handler.setFormatter(formatter)
    listener.start()

    if app is not None:
        app.register_listener(listener.stop, 'after_server_stop')
```

I guess the best thing is to either remove those logs or make sure they're enabled explicitly.

**Comment by chris-rl** [comment]
Additional information:
The actual debug log takes place in this lib: `urllib3.connectionpool`

**Comment by bahlo** [comment]
Fixed in #26, we'll cut a release this week

**Comment by bahlo** [comment]
@chris-rl We just released `v0.1.0-beta.4` which has the fix.
